submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/assertions/AllInOneSubmissionHelper.kt

```diff
@@ -6,33 +6,85 @@ import ac.uk.ebi.biostd.itest.factory.assertAllInOneSubmissionTsv
 import ac.uk.ebi.biostd.itest.factory.assertAllInOneSubmissionXml
 import ac.uk.ebi.biostd.persistence.common.service.SubmissionQueryService
 import ebi.ac.uk.extended.mapping.to.toSimpleSubmission
+import ebi.ac.uk.extended.model.ExtFile
 import ebi.ac.uk.extended.model.ExtProcessingStatus
 import ebi.ac.uk.extended.model.ExtSubmission
 import ebi.ac.uk.extended.model.ExtSubmissionMethod
+import ebi.ac.uk.extended.model.NfsFile
+import ebi.ac.uk.util.collections.second
+import ebi.ac.uk.util.collections.third
 import org.assertj.core.api.Assertions.assertThat
+import java.io.File
 import java.nio.file.Paths
 
 internal class AllInOneSubmissionHelper(
     private val submissionPath: String,
     private val submissionRepository: SubmissionQueryService
 ) {
-    internal fun assertSavedSubmission(accNo: String, method: ExtSubmissionMethod = ExtSubmissionMethod.PAGE_TAB) {
+
+    internal fun assertSavedSubmission(
+        accNo: String,
+        method: ExtSubmissionMethod = ExtSubmissionMethod.PAGE_TAB,
+        checkTabFiles: Boolean
+    ) {
         val extendedSubmission = submissionRepository.getExtByAccNo(accNo)
         assertThat(extendedSubmission.status).isEqualTo(ExtProcessingStatus.PROCESSED)
         assertThat(extendedSubmission.method).isEqualTo(method)
         assertThat(extendedSubmission.toSimpleSubmission()).isEqualTo(allInOneSubmission(accNo))
-        assertSubmissionFiles(extendedSubmission)
+        assertSubmissionFiles(extendedSubmission, checkTabFiles)
     }
 
-    private fun assertSubmissionFiles(submission: ExtSubmission) {
+    private fun assertSubmissionFiles(submission: ExtSubmission, checkTabFiles: Boolean) {
```

Comment:
As a wise friend once told me, boolean flags in methods aren't usually a good idea. Maybe it's better to have separated methods or just a method for specifically checking the page tab files and call it whenever necessary.

Also, I'd suggest `checkPageTabFiles` to keep consistency.